server.js

```diff
@@ -48,22 +91,22 @@ let notFound = (request, response) => {
 };
 
 let routes = [
-  { method: 'DELETE', path: /^\/users\/([0-9]+)$/, handler: deleteUser },
+  // { method: 'DELETE', path: /^\/users\/([0-9]+)$/, handler: deleteUser },
   { method: 'GET', path: /^\/users\/([0-9]+)$/, handler: getUser },
-  { method: 'PUT', path: /^\/users\/([0-9]+)$/, handler: putUser },
+  // { method: 'PUT', path: /^\/users\/([0-9]+)$/, handler: putUser },
   { method: 'GET', path: /^\/users\/?$/, handler: getUsers },
-  { method: 'POST', path: /^\/users\/?$/, handler: postUser },
-  { method: 'POST', path: /^\/signin\/?$/, handler: signIn },
-  { method: 'DELETE', path: /^\/articles\/([0-9]+)$/, handler: deleteArticle },
+  // { method: 'POST', path: /^\/users\/?$/, handler: postUser },
+  // { method: 'POST', path: /^\/signin\/?$/, handler: signIn },
+  // { method: 'DELETE', path: /^\/articles\/([0-9]+)$/, handler: deleteArticle },
   { method: 'GET', path: /^\/articles\/([0-9]+)$/, handler: getArticle },
-  { method: 'PUT', path: /^\/articles\/([0-9]+)$/, handler: putArticle },
+  // { method: 'PUT', path: /^\/articles\/([0-9]+)$/, handler: putArticle },
```

Comment:
Will we need a Put method for the articles as we won't be editing the articles, only displaying them via Post , Get and Delete?